docs/_static/discovery/v1.yaml

```diff
@@ -215,18 +221,41 @@ components:
         - credential_subject_id
         - vp
         - fields
+        - registrationParameters
       properties:
         id:
           type: string
           description: The ID of the Verifiable Presentation.
         credential_subject_id:
           type: string
           description: The ID of the Verifiable Credential subject (holder), typically a DID.
+        registrationParameters:
```

Comment:
Why not have it extracted through the Service Discovery definition, just like the other claims? Then you can address it as "auth_server_url", for instance.

Reply:
because you always have to add all registered params in the PD. You can still do this, this just helps in getting it all. Very handy for the `authServerURL` since it's always present. Adding that to the PD every time is annoying.